test/suites-ostree/backup-prune.robot

```diff
@@ -20,22 +20,19 @@ ${UNKOWN_DIR_REG}       unknown_fake*
 
 
 *** Test Cases ***
-Rebooting Healthy System Should Result In Pruning Old Backups
-    [Documentation]    Check that MicroShift backup logic prunes old backups
+Rebooting Healthy System Should Result In Pruning Old Backups And Keeping Unknown Directories
+    [Documentation]    Check that MicroShift backup logic prunes old backups and keeps unknown directories
 
     Create Fake Backup Directories
     Validate Fake Backups Exist
-    Reboot MicroShift Host
-    Wait For MicroShift Service
-    Validate Fake Backups Have Been Deleted
-
-Rebooting Healthy System Should Not Delete Unknown Directories
-    [Documentation]    Check that MicroShift backup logic does not delete unknown directories
 
-    Create Fake Unknown Backup Directories
+    Create Unknown Fake Backup Directories
     Validate Unknown Fake Backups Exist
+
     Reboot MicroShift Host
-    Wait For MicroShift Service
+    Wait For Healthy System
```

Comment:
Another way to solve this would be to move these steps into the suite setup. Then it would happen 1 time before each test case, and the test case could just check the expected results.

Reply:
That makes sense, I like it. I refactored but kept it as one test still, that seems more valid to me since it's a single prune action and we're not validating different reboot cycles to prune. What do you think?